main.tf

```
@@ -365,3 +365,30 @@ module "heat-cfn" {
     api_service = "heat-api-cfn"
   }
 }
+
+module "mysql-barbican" {
+  count      = var.enable-barbican ? (var.many-mysql ? 1 : 0) : 0
+  source     = "./modules/mysql"
+  model      = juju_model.sunbeam.name
+  name       = "mysql"
+  channel    = var.mysql-channel
+  scale      = var.ha-scale
+  many-mysql = var.many-mysql
+  services   = ["barbican"]
+}
+
+module "barbican" {
+  count                = var.enable-barbican ? 1 : 0
+  source               = "./modules/openstack-api"
+  charm                = "barbican-k8s"
+  name                 = "barbican"
+  model                = juju_model.sunbeam.name
+  channel              = var.openstack-channel
```

Open as draft until 2023.1 track is populated
```suggestion
  channel              = var.openstack-channel
```